### PR TITLE
Add ppc64le to build

### PR DESCRIPTION
**Pull request by jglaser**
Tentatively re-enable the ppc64le build, as suggested by @xhochy 

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jglaser** [comment]
@conda-forge-admin, please rerender

File: conda-forge.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # linux_ppc64le is explicitly removed again, see https://github.com/conda-forge/arrow-cpp-feedstock/issues/143
-provider: {linux_aarch64: default, win: azure}
+provider: {linux_aarch64: default, linux_ppc64le: azure, win: azure}
```
Comment (jakirkham) [inline conda-forge.yml:2]: So using `azure` means it is cross-compiled. `default` is built natively on Travis CI. Would suggest trying `default`, but we can go with cross-compiled if that doesn't work.

**Comment by jglaser** [comment]
@conda-forge-admin, please rerender

**Comment by jglaser** [comment]
Looks like this built correctly. The PR is therefore ready for review. I plan to enable CUDA builds on ppc64le in a separate PR.

**Comment by jakirkham** [comment]
> Looks like this built correctly.

Great, thanks for working on this! 😄 

> The PR is therefore ready for review. I plan to enable CUDA builds on ppc64le in a separate PR.

We don't quite have enough of the pieces in place to do that, but we are close.

Currently we have [a Docker image]( https://github.com/conda-forge/docker-images/tree/91ced4f0f698daa53cf50a6516c846f97171b5dd/linux-anvil-ppc64le-cuda ). Also we recently added [`cudatoolkit` to conda-forge]( https://github.com/conda-forge/cudatoolkit-feedstock ). We currently have built packages for CUDA 11.0, 10.2, and 10.1. Though there were issues building `cudatoolkit` for 10.1 ( https://github.com/conda-forge/cudatoolkit-feedstock/issues/12 )

We still need `cudatoolkit` packages for 10.0 and 9.2. Once that is done we need builds of [the `nvcc` wrapper package]( https://github.com/conda-forge/nvcc-feedstock ). We will also need to extend `conda-forge-pinning` to handle this architecture as we did before ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/285 ). Though these last two pieces should be more straightforward.

**Pull request merged**